CephFS PVC snapshot restore tests fails due to timeout

Test case failed because the the PVC creation from snapshot was delayed.

```
E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource restore-pvc-test-f1cde95c57474d2dafe6d29 describe output: Name:          restore-pvc-test-f1cde95c57474d2dafe6d29
E           Namespace:     namespace-test-9b3a4aafa1d14ee2a3ba7bf74
E           StorageClass:  ocs-storagecluster-cephfs
E           Status:        Pending
E           Volume:
E           Labels:        
E           Annotations:   volume.beta.kubernetes.io/storage-provisioner: openshift-storage.cephfs.csi.ceph.com
E                          volume.kubernetes.io/storage-provisioner: openshift-storage.cephfs.csi.ceph.com
E           Finalizers:    [kubernetes.io/pvc-protection]
E           Capacity:
E           Access Modes:
E           VolumeMode:    Filesystem
E           DataSource:
E             APIGroup:  snapshot.storage.k8s.io
E             Kind:      VolumeSnapshot
E             Name:      snapshot-test-164f523e33ae4026a284108ec7
E           Used By:     
E           Events:
E             Type     Reason                Age                From                                                                                                                      Message
E             ----     ------                ----               ----                                                                                                                      -------
E             Warning  ProvisioningFailed    61s                openshift-storage.cephfs.csi.ceph.com_csi-cephfsplugin-provisioner-84f4497b67-gx7tm_b74e1f3f-d9c6-4be2-86fb-9a1e52785a9a  failed to provision volume with StorageClass "ocs-storagecluster-cephfs": rpc error: code = Aborted desc = clone from snapshot is pending
E             Normal   Provisioning          23s (x7 over 63s)  openshift-storage.cephfs.csi.ceph.com_csi-cephfsplugin-provisioner-84f4497b67-gx7tm_b74e1f3f-d9c6-4be2-86fb-9a1e52785a9a  External provisioner is provisioning volume for claim "namespace-test-9b3a4aafa1d14ee2a3ba7bf74/restore-pvc-test-f1cde95c57474d2dafe6d29"
E             Warning  ProvisioningFailed    23s (x6 over 55s)  openshift-storage.cephfs.csi.ceph.com_csi-cephfsplugin-provisioner-84f4497b67-gx7tm_b74e1f3f-d9c6-4be2-86fb-9a1e52785a9a  failed to provision volume with StorageClass "ocs-storagecluster-cephfs": rpc error: code = Aborted desc = clone from snapshot is already in progress
E             Normal   ExternalProvisioning  2s (x7 over 63s)   persistentvolume-controller                                                                                               Waiting for a volume to be created either by the external provisioner 'openshift-storage.cephfs.csi.ceph.com' or manually by the system administrator. If volume creation is delayed, please verify that the provisioner is running and correctly registered.



ocs_ci/helpers/helpers.py:132: ResourceWrongStatusException
```

Test case is tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py::TestPvcSnapshot::test_pvc_snapshot[CephFileSystem]

--- a/tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py
+++ b/tests/functional/pv/pvc_snapshot/test_pvc_snapshot.py
@@ -136,7 +136,9 @@ class TestPvcSnapshot(ManageTest):
             pvc_name=restore_pvc_name,
             restore_pvc_yaml=restore_pvc_yaml,
         )
-        helpers.wait_for_resource_state(restore_pvc_obj, constants.STATUS_BOUND)
+        helpers.wait_for_resource_state(
+            restore_pvc_obj, constants.STATUS_BOUND, timeout=180
+        )
         restore_pvc_obj.reload()
         teardown_factory(restore_pvc_obj)
 

